README.md

```diff
@@ -10,35 +10,114 @@ Aims to wrap the Ticketmaster API with coverage for all Open Platform endpoints,
 
 ## System Requirements
 
- - NodeJS (v0.10 or greater)
+ - [NodeJS](https://nodejs.org) (v0.10 or greater)
 
-## Usage
+## Instalation:
 
+```bash
+npm install --save ticketmaster
+```
+
+## Client:
 
-1. Include the ticketmaster npm package:
+For browser usage there are two files in **dist/** folder
+```bash
+./dist/ticketmaster-client-[version].js (raw with source-maps)  
+./dist/ticketmaster-client-[version].min.js (minified)
+```
 
- - `npm install --save ticketmaster`
+Include one of them in to your project:
+```html
+...
+<script src="ticketmaster-client-[version].js"></script>
+<script src="ticketmaster-client-[version].min.js"></script>
+...
+```
 
-2. Require the package and make an API call:
+Use global variable **TMAPI** to make an API call (name can be changed in webpack settings during rebuild):
+
+```javascript
+TMAPI('your-api-key').discovery.v2.event.all()
+.then(function(result) {
+  // "result" is an object of Ticketmaster events information
+});
+```
+
+## Server:
+
+Require the package and make an API call:
 
 ```javascript
 var TM = require('ticketmaster');
 TM('your-api-key').discovery.v2.event.all()
-.then(function(events) {
-  // "events" is an array of Ticketmaster event information
-  console.log(events);
+.then(function(result) {
+  // "result" is an object of Ticketmaster events information
 });
 ```
 
 Alternative syntax if you are only interested in a subset of the API:
+
 ```javascript
 var EventAPI = require('ticketmaster').discovery.v2.event;
 EventAPI('your-api-key').all()
 ```
 
+## Rebuild source:
+
+In case you want to build your own bundle for client   
+
+`1`. Clone this repository  
+
+```bash
+git clone git@github.com:ticketmaster-api/sdk-javascript.git
+```
+`2`. install dependencies  
+
+```bash
+npm install
+```
+`3`. Run npm script: 
+
+- for raw (with source-maps) version of client lib use:
+```bash
+npm run-script dev
+```
+- for minified version of client lib use:
+```bash
+npm run-script prod
+```
+- or (for Windows users):
+```bash
+npm run-script win-prod
+```
+
+
+## Error handling:
+
+**Be aware:** no **.cath()** method provided! You should write it by your own.
```

Comment:
=)

2016-07-28 15:41 GMT+03:00 Adam Meghji <notifications@github.com>:

> In README.md
> <https://github.com/ticketmaster-api/sdk-javascript/pull/9#discussion_r72613995>
> :
>
> > +```bash
> > +npm run-script dev
> > +```
> > +- for minified version of client lib use:
> > +```bash
> > +npm run-script prod
> > +```
> > +- or (for Windows users):
> > +```bash
> > +npm run-script win-prod
> > +```
> > +
> > +
> > +## Error handling:
> > +
> > +**Be aware:** no **.cath()** method provided! You should write it by your own.
>
> Spelling: catch
>
> —
> You are receiving this because you were mentioned.
> Reply to this email directly, view it on GitHub
> <https://github.com/ticketmaster-api/sdk-javascript/pull/9/files/33b7c5395853843daf2fd681c523354e17287900#r72613995>,
> or mute the thread
> <https://github.com/notifications/unsubscribe-auth/AIxbjFzLDwiDK7613iD_i9eW0TBrWUoxks5qaKOAgaJpZM4JXFCu>
> .
>



-- 

*З повагою,*
*Павло Комеда*
*тел. +38(093)68-86-002*